app/ui/lib/Message.tsx

```diff
@@ -73,22 +76,29 @@ export const Message = ({
   variant = 'success',
   cta,
   icon,
+  hideableKey,
 }: MessageProps) => {
+  const [hidden, setHidden] = useState(false)
+  const closedMessages = localStorage.getItem('closedMessages')
```

Comment:
There's kind of a lot going on here in the implementation, but the calling API here is really nice. If we plan on doing more with localStorage it might be worth getting this or similar in place.

https://usehooks-ts.com/react-hook/use-local-storage#hook